openshift.bundle.yaml

```diff
@@ -114,7 +114,7 @@ spec:
     served: true
     storage: true
 ---
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
```

Comment:
are you sure? won't this break 3.11? do we need a separate bundle for 3.11? can't recall if we're calling that out